resources/src/operation/mod.rs

```diff
@@ -0,0 +1,220 @@
+use amount::Amount;
+use asset::AssetIdentifier;
+use serde::{de, Deserialize, Deserializer};
+use offer::PriceRatio;
+mod create_account;
+use self::create_account::CreateAccountFields;
+
+/// Operations are objects that represent a desired change to the ledger: payments, offers to
+/// exchange currency, changes made to account options, etc. Operations are submitted to the
+/// Stellar network grouped in a Transaction.
+
+#[derive(Debug, Serialize)]
+pub struct Operation {
+    id: i64,
+    paging_token: String,
+    type_i: u32,
```

Comment:
We should be able to map an actual enum to this instead of just the same integer? Or just omit it and serialize to it via a match against the type.